### PR TITLE
Toolbar night mode support

### PR DESCRIPTION
**Pull request by prateek-singh-3212**
## Purpose / Description
 Added toolbar support for night mode.

## Fixes
 Fixes issue #7832 

## Approach

-  I changed the toolbar background color to light grey or dark grey depending on the theme.

-  I have created a method that changes the icon color to white or black depending on the theme.

-  I have created a new drawable file to add a custom tag in the toolbar icon (White Color) which I am using in dark and black mode.

## How Has This Been Tested?
Tested on Samsung Galaxy M51, android 11 (Samsung One UI 3.1)
 
## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -52,18 +52,9 @@
     <JetCodeStyleSettings>
       <option name="PACKAGES_TO_USE_STAR_IMPORTS">
         <value>
-          <package name="java.util" alias="false" withSubpackages="false" />
```
Comment (david-allison-1) [inline .idea/codeStyles/Project.xml:55]: Could you revert this, not relevant to the PR

File: AnkiDroid/src/main/java/com/ichi2/anki/noteeditor/Toolbar.java [hunk 2]
```diff
@@ -306,6 +310,13 @@ public void onFormat(TextFormatter formatter) {
         mFormatCallback.performFormat(formatter);
     }
 
+    public void setIconColor(@ColorInt int color) {
+        for (int i=0; i < this.mToolbar.getChildCount(); i++) {
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/noteeditor/Toolbar.java:314]:
```suggestion
        for (int i = 0; i < this.mToolbar.getChildCount(); i++) {
```

File: AnkiDroid/src/main/java/com/ichi2/anki/NoteEditor.java [hunk 3]
```diff
@@ -1757,6 +1758,11 @@ private void initFieldEditText(FieldEditText editText, final int index, boolean
                 }
             });
         }
+
+        // Sets the background color of disabled EditText.
+        if (!enabled) {
+            editText.setBackgroundColor(Themes.getColorFromAttr(NoteEditor.this,R.attr.editTextBackgroundColor));
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/NoteEditor.java:1764]:
```suggestion
            editText.setBackgroundColor(Themes.getColorFromAttr(NoteEditor.this, R.attr.editTextBackgroundColor));
```

File: AnkiDroid/src/main/res/drawable/ic_add_toolbar_white_icon.xml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
```
Comment (david-allison-1) [inline AnkiDroid/src/main/res/drawable/ic_add_toolbar_white_icon.xml:2]: Are you able to use a tint here, rather than cloning the icon?

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
Looks good! minor questions

* Could you rebase this to remove the merge commits from the commit history
* Could you include screenshots from the 4 themes in the main post

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (prateek-singh-3212) [reply .idea/codeStyles/Project.xml:55]: >     * in the

Yeah, Sure

Comment (prateek-singh-3212) [reply .idea/codeStyles/Project.xml:55]: Yeah, Sure. Sorry for the inconvenience

**Comment by mrudultora** [comment]
Hi, @prateek-singh-3212 There are merge conflicts. Your fork is 95 commits behind the main repo. Please resolve them. Also, make your fork even with upstream.

File: AnkiDroid/src/main/res/drawable/ic_add_toolbar_white_icon.xml (hunk 4 above)
Comment (prateek-singh-3212) [reply AnkiDroid/src/main/res/drawable/ic_add_toolbar_white_icon.xml:2]: Yeah, it's working fine with the tint.

**Pull request closed without merging**